src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowSnykScanListenerLS.kt

```diff
@@ -155,45 +162,101 @@ class SnykToolWindowSnykScanListenerLS(
             ossResultsCount = ossResultsCount,
             iacResultsCount = iacResultsCount,
             containerResultsCount = containerResultsCount,
-            addHMLPostfix = rootNodePostFix
+            addHMLPostfix = rootNodePostFix,
         )
 
         snykToolWindowPanel.smartReloadRootNode(
             rootNode,
             userObjectsForExpandedNodes,
-            selectedNodeUserObject
+            selectedNodeUserObject,
         )
     }
 
-    private fun displayResultsForRootTreeNode(
+    private fun addInfoTreeNodes(
         rootNode: DefaultMutableTreeNode,
-        issues: Map<SnykFile, List<ScanIssue>>
+        issues: List<ScanIssue>,
+        securityIssuesCount: Int? = null,
+        fixableIssuesCount: Int? = null,
     ) {
-        fun navigateToSource(virtualFile: VirtualFile, textRange: TextRange): () -> Unit = {
-            io.snyk.plugin.navigateToSource(project, virtualFile, textRange.startOffset, textRange.endOffset)
+        // only add these info tree nodes to Snyk Code security vulnerabilities for now
```

Comment:
I made some choices here to only add these features for conistent ignores for Snyk Code. Mostly because I realised that in VSCode we render different messages for the different product types and rather than trying to add too many changes in this PR for other products, I'd rather open a feature ticket for the other product types.

If we want to spend time implementing this for the other product types then I can definitely do that, open to other opinions!

Reply:
Sounds sensible!